### DISCUSSION TITLE
Seplos 2.0 charge and discharge kWh data

### DISCUSSION THREAD

**Discussion by catweazle0172** [mr-manuel/venus-os_dbus-serialbattery#89] [Ideas]
Hello,

Would at be possible to collect the Charged kWh and Discharged kWh from the Seplos 2.0 16E version?
This data is available if you connect it by the CAN communication to a Victron system, but this data gets incorrect if you connect more than 1 battery.
It would be really nice to collect this data over the RS485 communication, so this can be readout true to Home Assistant for the Energy dashboard.

Thanks in advance!
Greetings,
Mike

**Comment by mr-manuel** [comment]
Could you organize and post the RS485 protocol here? Are you using Seplos or Seplos v3?

**Reply by catweazle0172** [reply]
Hi Mr-Manual,

I did had a look at a Seplos Modbus Communication protocol. (https://forum.iobroker.net/assets/uploads/files/1631599969177-seplos-48v-100a-bms-rs485-protocol.pdf)
But it looks like that this information is not shared over the RS485 connection unfortunately.
So there will be no way to collect this data...

Thanks anyway.
